### DISCUSSION TITLE
Clarification on response voting and privacy

### DISCUSSION THREAD

**Discussion by SkyyySi** [open-webui/open-webui#373] [General]
I couldn't find any information on how the response votings (thumbs up / down) are being used. Do they submit the response to some data pool like ShareGPT? Are they just there to mimic ChatGPT? I'd like to know what they do.

**Comment by tjbck** [comment]
Hi! We prioritize your privacy and data security above all. The message rating feature exists to facilitate users to generate RLHF dataset and everything is saved locally, your data doesn't get sent anywhere!  If you have any additional questions, do not hesitate to ask, Thanks!

**Reply by SkyyySi** [reply]
I see, thank you for the clarification. This would then lead me to my next question: Where can I find that data? I'd like to be able to contribute my conversations to public data lakes.

(Although, I guess it would make more sense to look at implementing data sharing support, but that's better discussed in an issue)

**Reply by tjbck** [reply]
<img width="650" alt="image" src="https://github.com/ollama-webui/ollama-webui/assets/25473318/71994a96-b025-4c90-8ea6-a70b7e14cdc7">
Here!
